### ISSUE TITLE
Help text on admin not working

### ISSUE THREAD

**Issue by helio-correia** [geex-arts/django-jet#213]
Hi there, I am using jet with an TabularInline and a help_text I can see the help text. The content is there but i've tried by hovering the arrow and the click and nothing works.

Thanks,
Hélio

![captura de ecra 2017-05-16 as 15 35 47](https://cloud.githubusercontent.com/assets/6276228/26111755/e54a358e-3a4d-11e7-8a1d-c41cf6a121dc.png)

![captura de ecra 2017-05-16 as 15 41 29](https://cloud.githubusercontent.com/assets/6276228/26111852/337e7918-3a4e-11e7-97c3-0e298958bea0.png)

**Comment by helio-correia** [comment]
Any news on this one?

**Comment by SalahAdDin** [comment]
Which version are you using?

**Comment by helio-correia** [comment]
Thanks for answering, i'm using the latest 1.0.6

**Comment by SalahAdDin** [comment]
Does this problem happen in the normal administration template?

**Comment by helio-correia** [comment]
When you say normal you are talking about the normal django or without the TabularInline? 
It's simple, the normal jet don't have tooltip, is the tooltip that don't work

**Comment by SalahAdDin** [comment]
Yes, normal django. Maybe we need to add this feauture or review what's the problem.

**Comment by Ismael-VC** [comment]
@helio-correia  please come to my Discord server so we can organize if you like:

  *   https://discord.gg/3yr5yeH

Welcome! 😄

**Comment by Ismael-VC** [comment]
Please come to the django-jet Discord server so we can organize if you like:

  *   https://discord.gg/3yr5yeH

Welcome! 😄

**Comment by maxicecilia** [comment]
I was able to fix this with 2 different approches

1. In templates/admin/edit_inline/tabular.html
Replace:
```
{% if field.help_text %}&nbsp;<img src="{% static "admin/img/icon-unknown.svg" %}" class="help help-tooltip" width="10" height="10" alt="({{ field.help_text|striptags }})" title="{{ field.help_text|striptags }}" />{% endif %}
```
with:
```
{% if field.help_text %}&nbsp;<span class="icon-question help help-tooltip" title="{{ field.help_text|striptags }}" style="color: #82b982;"></span>{% endif %}
```
This was a convenient hack because I already had a local copy of tabular.html

2. In static/jet/js/src/layout-updaters/icons.js
Replace:
```
$('img[src$="admin/img/icon-unknown.gif"]').add('img[src$="admin/img/icon-unknown.svg"]').after($('<span class="icon-question">'));
```
with
```
$('img[src$="admin/img/icon-unknown.gif"]').add('img[src$="admin/img/icon-unknown.svg"]').after($('<span class="help help-tooltip icon-question" title="' + $($('img[src$="admin/img/icon-unknown.svg"]')[0]).attr('title') + '" >'));
```
The second approach is probably the right one, but ff course, this requires to rebuild the js bundle, and I really didn't like the code, so I stuck with approach 1.
If you think the second approach is good enough, I can push a pull request.

**Comment by SalahAdDin** [comment]
@helio-correia Can you test the second approach?
If it works i will make a PR and merge it in my fork.

**Comment by helio-correia** [comment]
I had some trouble running with gulp, but I test the second option with browser console and it works like a charm. If you want I can test with more time, but i think its ready to go

**Comment by SalahAdDin** [comment]
@helio-correia Please test it a few and after make PR 👍

**Comment by maxicecilia** [comment]
@helio-correia I was able to create the bundle.min.js with gulp, check it out in #245
